packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-vector-settings.tsx

```diff
@@ -67,7 +70,12 @@ const LockedVectorSettings = (props: Props) => {
             visiblelabel += ` ${labels.map((l) => l.text).join(", ")}`;
         }
 
-        const str = `Vector${visiblelabel} from (${tail[0]}, ${tail[1]}) to (${tip[0]}, ${tip[1]})`;
+        let str = `Vector${visiblelabel} from (${tail[0]}, ${tail[1]}) to (${tip[0]}, ${tip[1]})`;
+
+        const vectorAppearance =
+            generateLockedFigureAppearanceDescription(lineColor);
+        str += `${vectorAppearance}`;
```

Comment:
This wrapping isn't necessary, since `generateLockedFigureAppearanceDescription` always returns a string.

```suggestion
        str += vectorAppearance;
```